Client/src/utils/TokenService.js

```
@@ -0,0 +1,342 @@
+    // Fill with cryptographically secure random values
+    window.crypto.getRandomValues(buffer);
+    
+    // Convert to a hex string and combine with timestamp for uniqueness
+    const randomHex = Array.from(buffer)
+      .map(b => b.toString(16).padStart(2, '0'))
+      .join('');
+      
+    // Include timestamp prefix for additional entropy and sortability
+    const timestamp = Date.now().toString(36);
+    
+    return timestamp + '-' + randomHex;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)